scripts/NER_block.py

```
@@ -0,0 +1,199 @@
+import re
```

Ruff (0.7.0)

`re` imported but unused

Remove unused import: `re`

(F401)